Missionframework/description.ext

```diff
@@ -151,7 +151,7 @@ allowFunctionsLog = 0;
 
 // As a security measure, functions are by default protected against rewriting during mission.
 // This restriction does not apply in missions previewed from the editor.
-allowFunctionsRecompile = 0;
+allowFunctionsRecompile = __EVAL([0, 1] select (is3DENMultiplayer || is3DEN));
```

Comment:
What about the comment above concerning not apply in mission preview from the editor? Wrong hint or just overlooked?

Reply:
In my experience it was working only in SP preview.

Reply:
Ok. As it won't hurt in that implementation with __EVAL, it's fine and we can leave it that way. 👍